cartridges/algolia_sg_changes/cartridge/templates/default/search/simplesearch.isml

```diff
@@ -6,7 +6,7 @@
         data-category="${pdict.cgid}"
         data-category-display-name-path="${pdict.categoryDisplayNamePath}"
         data-category-display-name-path-separator="${pdict.categoryDisplayNamePathSeparator}"
-        data-q="${pdict.q}"
+        data-q="${empty(pdict.q) ? '' : pdict.q}"
```

Comment:
Where is this `empty` function coming from? jQuery?

Reply:
No, it's SFCC-specific. It's a global operator, can be accessed from anywhere without importing anything.
https://documentation.b2c.commercecloud.salesforce.com/DOC1/index.jsp?topic=%2Fcom.demandware.dochelp%2FScriptProgramming%2FDemandwareJavaScriptExpressionsinISML.html